src/frontend.cpp

```diff
@@ -12,13 +12,21 @@ void Frontend::WriteFile() {
 	std::cout << "Output written to " << outputFileName << std::endl;
 }
 
-void Frontend::Helper(Error errorCode) {
+void Frontend::Exception(Error errorCode, const std::string &input) {
 	if (errorCode == fileError) {
 		std::cout << "Error: No file for parsing" << std::endl;
+	} else if (errorCode == argError) {
+		std::cout << "Invalid command-line argument used " << input << '\n';
+		std::cout << "Please see -h for more info" << std::endl;
+	}
+}
+
+void Frontend::Helper(Error errorCode) {
+	if (errorCode == helpArgument) {
```

Comment:
Maybe just remove this if statement, and rename the function to `PrintHelp`?

Reply:
If we need to make more helpers for different arguments in the command-line the we can put them in the helper method.

Reply:
Well, but until then, let's not overcomplicate things